docs/Drawing.md

```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
       .Ellipse(256, 96, 192, 8, 0, 360)
       .Draw(image);
 }
-```
\ No newline at end of file
+```
+
+## Adding Text To Existing Image
+
+![Example of adding text to existing image](./img/example_addTextToExistingImage.jpg)
+
+```C#
+var pathToBackgroundImage = "path/to/background.png";
+var pathToNewImage = "path/to/newImage.png";
+var textToWrite = "Insert This Text Into Image";
+
+/* These settings will create a new caption
+    which automatically resizes the text to best
+    fit within the box.
+*/
+var readSettings = new MagickReadSettings()
+{
+    Font = "Calibri",
+    TextGravity = Gravity.Center,
+    BackgroundColor = MagickColors.Transparent,
+    Height = 250, // height of text box
+    Width = 680 // width of text box
+};
+
+using (var image = new MagickImage(pathToBackgroundImage))
```

Comment:
Could you not "layer" the usings and add braces?

Reply:
👍